src/main/java/core/basesyntax/MyLinkedList.java

```diff
@@ -3,48 +3,156 @@
 import java.util.List;
 
 public class MyLinkedList<T> implements MyLinkedListInterface<T> {
+    int size;
+    Node<T> head;
+    Node<T> tail;
+
+    public MyLinkedList() {
+        size = 0;
```

Comment:
if you want you can remove initialization from the constructor because `int` value by default has 0 value.